app/views/home.html

```diff
@@ -1,156 +1,87 @@
-        <div class="govuk-grid-row govuk-!-margin-bottom-6 home-statuses__flex-row">
-          {% set statuses = ['EYTS recommended', 'EYTS awarded', 'Deferred', 'Withdrawn'] %}
-          {% for status in statuses %}
-            <div class="govuk-grid-column-one-quarter">
-              {{ statusCard(records, status) }}
-            </div>
-          {% endfor %}
-        </div>
-      {% else %}
-        <div class="govuk-grid-row govuk-!-margin-bottom-6 home-statuses__flex-row">
-          <div class="govuk-grid-column-one-quarter">
-            <a href="/records?filterStatus=EYTS recommended&filterStatus=QTS recommended" class="status-card status-card--recommended">
-              {% set recordCount = records | where("status", ['EYTS recommended','QTS recommended']) | length %}
-              <span class="status-card__count">{{recordCount}}</span>
-              <span class="status-card__status">Qualification recommended</span> <span class="govuk-visually-hidden"> records. View these records.</span>
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+      <h2 class="govuk-heading-l">Registered trainees</h2>
+      <p class="govuk-body">
+        <a class="govuk-link" href="/records?filterCycle=_unchecked">See all registered trainees ({{ registeredTrainees | length }})</a>
+      </p>
+      <h3 class="govuk-heading-m">
+        2020 to 2021 trainees
+        <span class="govuk-body">
+           —  <a class="govuk-link" href="/records?filterCycle=2020+to+2021">see all 
+            <span class="govuk-visually-hidden">
+              2020 to 2021 trainees
+            </span> 
+            ({{ registeredTrainees | filterByYear(["2020 to 2021"]) | length }})
             </a>
-          </div>
-          <div class="govuk-grid-column-one-quarter">
-            <a href="/records?filterStatus=EYTS awarded&filterStatus=QTS awarded" class="status-card status-card--awarded">
-              {% set recordCount = records | where("status", ['EYTS awarded','QTS awarded']) | length %}
-              <span class="status-card__count">{{recordCount}}</span>
-              <span class="status-card__status">Qualification awarded</span><span class="govuk-visually-hidden"> records. View these records.</span>
-            </a>
-          </div>
-          {% set statuses = ['Deferred', 'Withdrawn'] %}
-          {% for status in statuses %}
-            <div class="govuk-grid-column-one-quarter">
-              {{ statusCard(records, status) }}
-            </div>
+        </span>
+      </h3>
+    </div>
+  </div>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <div class="app-home-statuses">
+
+          {% set statuses = ['Pending TRN', 'TRN received', 'Recommended', 'Awarded', 'Deferred', 'Withdrawn'] %}
+          {% for status in statuses  %}
+          {% set statusString = status %}
+            {% if status == 'Recommended' %}
+                {% set searchStatus = ['EYTS recommended', 'QTS recommended'] %}
```

Comment:
I think to get the filters working this could do
```suggestion
                {% set searchStatus = "EYTS recommended&filterStatus=QTS recommended" %}
```